packages/developer-portal/src/components/pages/app-detail/app-revision-modal/app-revision-modal.tsx

```diff
@@ -107,12 +112,15 @@ export const DeveloperAppRevisionModal: React.FC<DeveloperAppRevisionModalProps>
 
   const revisionsData = revisions?.data
   const latestAppRevisionId = revisionsData && revisionsData[0].id
-  const { declineFormState, revisionDetailData } = revisionDetailState
+  const { isLoading, data } = revisionDetailState
+
+  // const isDeclining = declineFormState === 'SUBMITTING'
+  // const isDeclinedSuccessfully = declineFormState === 'SUCCESS'
+  const isDeclining = false
+  const isDeclinedSuccessfully = false
```

Comment:
I'll work on this later in the next PR when refactoring the `declineAppRevisionState`